DTXManiaプロジェクト/コード/全体/CSkin.cs

```diff
@@ -98,41 +100,23 @@ public int n位置_次に鳴るサウンド
 						sound.n位置 = value;
 				}
 			}
-			public int n音量_現在のサウンド
+			public int nAutomationLevel_現在のサウンド
 			{
 				get
 				{
 					CSound sound = this.rSound[ 1 - this.n次に鳴るサウンド番号 ];
 					if( sound == null )
 						return 0;
 
-					return sound.n音量;
+					return sound.AutomationLevel;
 				}
 				set
 				{
 					CSound sound = this.rSound[ 1 - this.n次に鳴るサウンド番号 ];
-					if( sound != null )
-						sound.n音量 = value;
-				}
-			}
-			public int n音量_次に鳴るサウンド
```

Comment:
Removal of an unused property with an underlying dependency on CSound volume information.